guides/hack/01-getting-started/05-input-and-output-examples/hsl-io-basics.hack

```diff
@@ -14,10 +14,12 @@ async function main_async(): Awaitable<void> {
   await $out->writeAsync($message);
 
   // copy to a temporary file, automatically closed at scope exit
-  await using ($f = File\temporary_file()) {
+  using ($tf = File\temporary_file()) {
+    $f = $tf->getHandle();
+
     await $f->writeAsync($message);
```

Comment:
actually, these should generally be writeAllAsync nowadays; this can partially succeed